servers/zms/src/main/java/com/yahoo/athenz/zms/ZMSImpl.java

```diff
@@ -2607,6 +2607,53 @@ public Template getTemplate(ResourceContext ctx, String templateName) {
         return template;
     }
 
+    @Override
+    public DomainTemplateDetailsList getDomainTemplateDetailsList(ResourceContext ctx, String domainName) {
+        DomainTemplateDetailsList domainTemplateDetailsList = null;
+        List<TemplateMetaData> templateMetaDataList;
```

Comment:
this one is not needed, please see my comment below.

Reply:
done.